gen/definitions/cts.yaml

```diff
@@ -2,6 +2,111 @@
 name: CTS
 path: Cisco-IOS-XE-native:native/cts
 doc_category: CTS
+test_tags: [C9000V]
 attributes:
   - yang_name: Cisco-IOS-XE-cts:authorization/list
     example: Tacacs-GROUP
+  - yang_name: Cisco-IOS-XE-cts:sgt
+    tf_name: sgt
+    example: 200
+  - yang_name: Cisco-IOS-XE-cts:sxp/sxp-def-enable
+    tf_name: sxp_enable
+    example: true
+  - yang_name: Cisco-IOS-XE-cts:sxp/default/password/type
+    tf_name: sxp_default_password_type
+    example: 0
+  - yang_name: Cisco-IOS-XE-cts:sxp/default/password/secret
+    tf_name: sxp_default_password_secret
```

Comment:
We only need to specific the `tf_name` if it is different than the one derived from the `yang_name`. In this case it is exactly the same, hence it can be omitted.